### ISSUE TITLE
Run continuous_blockchain test on 30 blocks

### ISSUE THREAD

**Issue by teor2345** [ZcashFoundation/zebra#1836]
**Is your feature request related to a problem? Please describe.**

Once #1414 is finished, we will have mainnet and testnet blocks `0-30` in our test vectors.

We also want to use these blocks in the `CheckpointVerifier` `continuous_blockchain` test.

**Describe the solution you'd like**

- [ ] Wait for #1414 to merge
- [ ] Use mainnet and testnet blocks `0-30` in the `continuous_blockchain` test
  - the `CONTINUOUS_MAINNET/TESTNET_BLOCKS` test vectors should do this automatically
- [ ] Modify the checkpoints in these tests to `0`, `10`, `19`, and `28`, and update the relevant assertions if needed

**Describe alternatives you've considered**

Do nothing.

The extra blocks probably don't gain us a lot of extra test coverage. So this change is a low priority.

**Additional context**

These tests were re-enabled in #1285.

**Comment by teor2345** [comment]
It was pretty easy to make these changes as part of #1837, so these tests should now automatically include any extra blocks.
